INSTALL.md

```diff
@@ -17,6 +19,41 @@ Table of Contents:
 
 6. [More SCons options](#6-more-scons-options)
 
+0. Overall summary
+==================
+
+While the sections below detail how to install GalSim including its required and
+optional dependencies, this section gives a brief summary.  A minimal
+installation of GalSim requires the following dependencies. This dependency list
+includes a canonical version number that is known to work. In most cases, other
+recent versions will also work:
+
+- Python (2.7, 3.4, 3.5)
+- SCons (2.1.0)
+- NumPy (1.11)
+- Astropy (1.1.1)
+- Future (0.16.0)
+- FFTW (3.3)
+- TMV (0.73)
+- Boost (1.61)
+
+A few optional dependencies provide additional functionality, but GalSim can
+otherwise be compiled and used without them.  Basic WCS functionality is native
+to GalSim, but for users with more complicated WCS needs, we recommend
+installing starlink-pyast. Astropy's WCS package is also supported, but note
+that it requires scipy as an additional dependency.  To use yaml for config
+parsing, the pyyaml module is needed.  Faster text file parsing for reading in
+bandpasses and SEDs can be enabled if you have the pandas module (but the code
+will work, albeit more slowly, without this module).
+
+There are alternative modules to enable some of the GalSim functionality, such
+as pyfits and so on (listed below as optional), but the above list is
+a minimal set of dependencies to get started.  The sections below give a lot
```

Comment:
Maybe remove this first sentence, since pyfits is kind of the only thing left that wasn't mentioned above.  (I think.)  And that's probably irrelevant for most users now I think.